Kitodo/src/main/webapp/WEB-INF/templates/includes/tasks/taskList.xhtml

```diff
@@ -65,21 +65,25 @@
         <p:column headerText="#{msgs.actions}">
             <h:form id="actions">
                 <!-- assign task -->
-                <h:commandLink id="take" action="#{CurrentTaskForm.takeOverTask}"
+                <h:commandLink id="take"
+                               actionListener="#{CommentForm.setProcessId(item.process.id)}"
+                               action="#{CurrentTaskForm.takeOverTask}"
                                rendered="#{(item.processingStatus == 'OPEN' and !item.batchStep) || (item.processingStatus == 'OPEN' and item.batchStep and !item.batchAvailable)}"
                                title="#{msgs.bearbeitungDiesesSchrittsUebernehmen}">
                     <i class="fa fa-bell-o"/>
                     <f:setPropertyActionListener value="#{item.id}" target="#{CurrentTaskForm.taskById}"/>
                 </h:commandLink>
                 <!-- already assigned task (this user) -->
-                <h:link outcome="/pages/currentTasksEdit"
-                        rendered="#{(item.processingStatus == 'INWORK' and item.processingUser.id == LoginForm.loggedUser.id and !item.batchStep) || (item.processingStatus == 'INWORK' and item.processingUser.id == LoginForm.loggedUser.id and item.batchStep and !item.batchAvailable)}"
-                        title="#{msgs.inBearbeitungDurch}: #{item.processingUser!=null and item.processingUser.id!=0 ? item.processingUser.fullName:''}">
+                <h:commandLink action="/pages/currentTasksEdit"
```

Comment:
Selenium tests fail, because link to '/kitodo/pages/currentTasksEdit.jsf?id=12' is not found. Link is not found, because link can't be used as an action. This change disable possibility to access currentTaskEdit page.